libnavui-dropin/api/current.txt

```diff
@@ -61,7 +61,7 @@ package com.mapbox.navigation.dropin {
   }
 
   public final class ViewBinderCustomization {
-    ctor public ViewBinderCustomization();
```

Comment:
Is this supposed to be removed? Doesn't it break API?

Reply:
This is a valid point. I have introduced a fix. Let me know if you have a better solution.